server/src/main/java/io/crate/replication/logical/metadata/pgcatalog/PgSubscriptionTable.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.replication.logical.metadata.pgcatalog;
+
+import io.crate.metadata.RelationName;
+import io.crate.metadata.SystemTable;
+import io.crate.metadata.pgcatalog.OidHash;
+import io.crate.metadata.pgcatalog.PgCatalogSchemaInfo;
+import io.crate.replication.logical.metadata.Subscription;
+
+import java.util.stream.Collectors;
+
+import static io.crate.types.DataTypes.BOOLEAN;
+import static io.crate.types.DataTypes.INTEGER;
+import static io.crate.types.DataTypes.STRING;
+import static io.crate.types.DataTypes.STRING_ARRAY;
+
+public class PgSubscriptionTable {
+
+    public static final RelationName IDENT = new RelationName(PgCatalogSchemaInfo.NAME, "pg_subscription");
+
+    public static SystemTable<SubscriptionRow> create() {
+        return SystemTable.<SubscriptionRow>builder(IDENT)
+            .add("oid", INTEGER, r -> OidHash.subscriptionOid(r.name, r.subscription))
+            .add("subdbid", INTEGER, r -> OidHash.subscriptionHostsOid(r.name, r.subscription))
+            .add("subname", STRING, r -> r.name)
+            .add("subowner", STRING, r -> r.subscription.owner())
+            .add("subenabled", BOOLEAN, r -> true)
+            .add("subbinary", BOOLEAN, r -> true)
+            .add("substream", BOOLEAN, r -> false)
```

Comment:
>If true, the subscription will allow streaming of in-progress transactions

put false since we agreed not to replicate `Translog` for now - but it needs attention.

Reply:
It should be `false` despite of any `translog` replication, as this is about transaction which we don't support at all.